hw04_lru_cache/go.mod

```
@@ -1,4 +1,4 @@
-module github.com/fixme_my_friend/hw04_lru_cache
+module github.com/BorisPlus/OTUS-Go-2023-03/tree/master/hw04_lru_cache
```

```suggestion
module github.com/BorisPlus/OTUS-Go-2023-03/hw04_lru_cache
```
`tree/master` - часть пути в UI-роутинге GitHub